hack/verify-test-code.sh

```diff
@@ -51,6 +51,30 @@ do
     fi
 done
 
+errors_expect_be_bool=()
+for file in "${all_e2e_files[@]}"
+do
+    if grep -E "gomega\.Expect\(.*\)\.(To|NotTo)\(gomega\.(BeTrue|BeFalse)\(\)" "${file}" > /dev/null
```

Comment:
do we need to have the `gomega` as part of the call line?
see here it's missing https://github.com/kubernetes/kubernetes/pull/86157/files#diff-40c7fe92055b0c9cbceaf4e26af9a740R39